app/component/SummaryRow.js

```diff
@@ -68,7 +68,7 @@ export const RouteLeg = ({ leg, large, intl }) => {
         text={message}
         className={cx('line', 'call')}
         vertical
-        withBar
+        // withBar
```

Comment:
Would it be possible to have this configurable? Are there already existing option mechanisms we could adapt?

Reply:
By default withBar parameter is false in the RouteNumber constructor. When it's given, it becomes true.